set.mm

```diff
@@ -476842,8 +476919,30 @@ The sumset operation can be used for both group (additive) operations and
     ${
       $d .X. a $.  $d .X. e i $.  $d B e i $.  $d E a e i $.  $d G e i $.
       $d I a e i $.  $d R e i $.  $d a ph $.  $d e i ph $.
-      ringlsmss2.1 $e |- ( ph -> E C_ B ) $.
-      ringlsmss2.2 $e |- ( ph -> I e. ( LIdeal ` R ) ) $.
+      ringlsmss1.1 $e |- ( ph -> R e. CRing ) $.
+      ringlsmss1.2 $e |- ( ph -> E C_ B ) $.
+      ringlsmss1.3 $e |- ( ph -> I e. ( LIdeal ` R ) ) $.
+      $( The product of an ideal ` I ` of a commutative ring ` R ` with some
+         set E is a subset of the ideal.  (Contributed by Thierry Arnoux,
```

Comment:
would it be more accurate to say ``a subset ` E ` ``? I might be missing something because it looks like ringlsmss1 can be directly proved from ringlsmss2, and since ringlsmss2's proof didnt change I would assume it doesnt use ringlsmss1.

Reply:
The big difference between ~ringlsmss1 and ~ringlsmss2 is that the former requires the ring to be commutative, while the latter does not.

With $I$ being a left-ideal and $E$ any set in the base set, $EI$ is always a left-ideal, but $IE$ is only an ideal if the ring is commutative. @savask noted that if the ring is commutative, we don't need to speak about left- or -right ideals anymore, since all ideals are two-sided by [crng2idl](https://us.metamath.org/mpeuni/crng2idl.html).